pycloak/middleware.py

```diff
@@ -184,3 +186,56 @@ def allow_default_login(self, request) -> bool:
 
     def get_token_id(self, request, jwt_data) -> str:
         return jwt_data[conf.TOKENID_CLAIM]
+
+    def store_claim_on_user(self, request, user):
+        for claim, attr in conf.CLAIM_TO_USER_MAPPING.items():
+            if not attr.get("field"):
+                raise ImproperlyConfigured(
+                    f"PYCLOAK_CLAIM_TO_USER_MAPPING: {claim} has no field specified"
+                )
+
+            # check for related fields on the user model
+            rel_fields = attr["field"].split(".")
+            field = rel_fields.pop(0)
```

Comment:
Pops the first item?

Reply:
Pops the last (https://docs.python.org/3/tutorial/datastructures.html):

> Remove the item at the given position in the list, and return it. If no index is specified, a.pop() removes and returns the last item in the list.